### ISSUE TITLE
_cjose_jws_verify_sig_rs256 always returns true

### ISSUE THREAD

**Issue by zandbelt** [cisco/cjose#10]
what's up with this line:
https://github.com/cisco/cjose/blob/master/src/jws.c#L827

````
static bool _cjose_jws_verify_sig_rs256(
            cjose_jws_t *jws, 
            const cjose_jwk_t *jwk, 
            cjose_err *err)
{
    return true;
...
````

**Comment by linuxwolf** [comment]
That's clearly a bug.  Possibly from a bad merge, although I can't recall any of those.

**Comment by zandbelt** [comment]
The code that is triggered now doesn't work conform spec, see #12

**Comment by linuxwolf** [comment]
Re-opening until #12 merges.

**Comment by linuxwolf** [comment]
Closed with merge of #12.
